docs/.vuepress/components/SegmentedBarLiveDemo.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <StackLayout>
+    <StackLayout style="margin-top:20px" class="demo-panel">
+      <Label text="Live example" class="demo-panel__title"/>
+      <SegmentedBar
+        @selectedIndexChanged="onIdxChanged($event)"
+        :selectedBackgroundColor="selectedTabColor"
+        v-model="selectedIndex"
+        style="padding-bottom:2em;"
+      >
+        <SegmentedBarItem v-for="(segment, index) in segments" :title="segment.name" :key="index"/>
+      </SegmentedBar>
+    </StackLayout>
+    <StackLayout class="demo-panel">
```

Comment:
Some margin could be added between buttons, child rows and index elements, so that the demo props can be more distinctive.

Reply:
Some margins are added to make element look more distinctive. 
![image](https://user-images.githubusercontent.com/812622/57303304-b51e6300-70e5-11e9-87fe-3e8ca2b13a87.png)